stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -189,6 +191,133 @@ class Embedding(Layer[tf.Tensor, tf.Tensor]):
         mask_zero: bool = False,
         input_length: int | None = None,
         trainable: bool = True,
+        dtype: _LayerDtype | None = None,
+        dynamic: bool = False,
+        name: str | None = None,
+    ) -> None: ...
+
+class Conv2D(Layer[tf.Tensor, tf.Tensor]):
+    def __init__(
+        self,
+        filters: int,
+        kernel_size: int | tuple[int, int],
```

Comment:
Iterable[int]. The actual implementation is more lax then strict tuple and I suspect think list like [3, 3] is pretty common. Same comment for rest of tuple[int, int], usually runtime normalizes iterable first.

Reply:
I checked an both example below are valid.

```python
tf.keras.layers.Conv2D(2, [3,3])
tf.keras.layers.Conv2D(2, iter([3,3]))
```

Fixed in 16c5775fd.